src/main/java/net/minecraftforge/fml/client/SplashProgress.java

```diff
@@ -169,6 +174,7 @@ public static void start()
         barBorderColor =     getHex("barBorder",     0xC0C0C0);
         barColor =           getHex("bar",           0xCB3D35);
         barBackgroundColor = getHex("barBackground", 0xFFFFFF);
+        memWarnPercent     = getInt("memoryWarningPercent", 85);
```

Comment:
Seeing how `barBorderColor`, `barColor`, and `barBackgroundColor` are spaced to the same location for the second parameter, they should be extended to start at the second parameter of `memWarnPercent` to be consistent.